planner/core/pom.xml

```diff
@@ -17,32 +16,106 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 -->
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
     <groupId>eu.seaclouds-project</groupId>
-    <artifactId>planner</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
-  </parent>
-  <artifactId>planner-core</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
-  <name>planner-core</name>
-  <url>http://maven.apache.org</url>
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
-  <dependencies>
-
-    <dependency>
-      <groupId>eu.seaclouds-project</groupId>
-      <artifactId>matchmaker</artifactId>
-      <version>0.1.0-SNAPSHOT</version>
-    </dependency>
-    <dependency>
-      <groupId>eu.seaclouds-project</groupId>
-      <artifactId>planner-utils</artifactId>
-      <version>0.1.0-SNAPSHOT</version>
-    </dependency>
-  </dependencies>
-</project>
+    <artifactId>core</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <parent>
+        <groupId>eu.seaclouds-project</groupId>
+        <artifactId>planner</artifactId>
+        <version>1.0-SNAPSHOT</version>
```

Comment:
About the version, I think that we should keep the same one along all the modules of the project. Since we don't have a complete guideline for versioning I don't know which pattern should we follow (`0.1.0-SNAPSHOT` vs `1.0-SNAPSHOT`). 

@andreaturli any ideas?